viewer/packages/360-images/src/annotation/PolygonAnnotationData.ts

```diff
@@ -0,0 +1,39 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { BufferGeometry, Matrix4, Shape, ShapeGeometry } from 'three';
+
+import { AnnotationsObjectDetection } from '@cognite/sdk';
+
+import { ImageAnnotationObjectData } from './ImageAnnotationData';
+
+export class PolygonAnnotationData implements ImageAnnotationObjectData {
+  private readonly _geometry: ShapeGeometry;
+
+  constructor(annotation: AnnotationsObjectDetection) {
+    this._geometry = this.createGeometry(annotation);
+  }
+
+  createGeometry(annotation: AnnotationsObjectDetection): ShapeGeometry {
+    const points = annotation.polygon!.vertices.map(({ x, y }) => ({ x: 0.5 - x, y: 0.5 - y }));
```

Comment:
Should handle annotation.polygon being undefined. Either handle it, or if it technically shouldn't be possible, then assert it.